4337/contracts/test/certora/Account.sol

```diff
@@ -0,0 +1,140 @@
+// SPDX-License-Identifier: LGPL-3.0-only
```

Comment:
Wdyt about including this in the certora folder directly, similar to how it's done in the safe-contracts repo?